packages/start-slicemachine/test/index.test.ts

```diff
@@ -0,0 +1,6 @@
+import { it, expect } from "vitest";
+
+// TODO: Dummy test, meant to be removed when real tests come in
+it("exports something", () => {
```

Comment:
🤔

Reply:
Indeed, we need to write tests for `start-slicemachine`. I was waiting for `@slicemachine/init` to be tested so we could copy some of its testing practices.